app/views/layouts/_navbar.html.erb

```diff
@@ -1,14 +1,20 @@
+<ul id="dropdown1" class="dropdown-content">
+  <% @categories.each do |category| %>
+    <li>
+      <%= link_to category.name, "/categories/#{category.name}" %>
+    </li>
+  <% end %>
+  <li class="divider"></li>
+  <li><a href='/categories'>All Categoies</a></li>
+</ul>
 <nav>
-
   <div class="nav-wrapper grey">
-
-
     <ul id="nav-mobile" class="left hide-on-med-and-down">
-			<li><a href="about.html" >About Us</a></li>
```

Comment:
moved this line into each of our _nav_helper methods in order to push it to the right side of the logo